src/components/Notifications/ToastNotification/ToastNotificationProvider.tsx

```diff
@@ -262,28 +271,38 @@ const ToastNotificationProvider: FC<PropsWithChildren<Props>> = ({
 
   const helper: ToastNotificationHelper = {
     notifications,
-    failure: (title, error, message, actions) =>
-      addNotification(failure(title, error, message, actions)),
-    info: (message, title, actions) =>
+    failure: (title, error, message, actions, entityId) =>
+      addNotification({
+        message: formatErrorMessage(message, error),
+        title,
+        error,
+        actions,
+        entityId,
+        type: NotificationSeverity.NEGATIVE,
+      }),
+    info: (message, title, actions, entityId) =>
```

Comment:
In my opinion, we should extend the failure helper with anything we need in here in `messageBuilder.tsx`. The other severity types like `info`, `success` and alike should rely on helpers over there as well instead of building the message ad hoc.

Reply:
I think if we have good reason to have an entity ID on the notification, it should be added to the base notification. Though I currently don't understand yet why a notification should have an entity ID. If it turns out to be needed, maybe this should even be an entity ID list, as a notification could relate to multiple entities.

Reply:
Maybe we and others concerned can have a quick chat on this?